src/playground/App.js

```diff
@@ -125,12 +125,33 @@ const App = () => {
         setOptions(newOptions);
         storeState({ newOptions });
     };
+    const [showConfigMenu, toggleConfigMenu] = useState(false);
+    const [isConfigHidden, toggleConfig] = useState(window.matchMedia("(min-width: 1023px)").matches);
```

Comment:
Let's use `const [var, setVar] = useState(false);` format? `toggleConfig` not seems fit here since we can use it to set `showConfigMenu` to any variable.

```suggestion
    const [showConfigMenu, setShowConfigMenu] = useState(false);
    const [isConfigHidden, setIsConfigHidden] = useState(window.matchMedia("(min-width: 1023px)").matches);
```